reference/promotions.v3.yml

```diff
@@ -1178,6 +1256,24 @@ components:
           example: '2019-01-20T22:00:00.000Z'
       required:
         - code
+
+    BulkCouponCode:
+      type: object
+      properties:
+        code:
+          type: string
+          pattern: '[a-zA-Z0-9_\ -]'
+          description: A unique code that can be used to manually apply a discount. Only letters, numbers, white spaces, underscores and hyphens are allowed.
+          example: TEST-COUPON-CODE
+          maxLength: 50
+          minLength: 12
```

Comment:
not in scope of this PR: probably we can define a type for this and reuse it in many places